client/src/main/java/io/split/engine/sse/dtos/GenericNotificationData.java

```diff
@@ -17,7 +22,10 @@ public GenericNotificationData (Long changeNumber,
                                     OccupancyMetrics occupancyMetrics,
                                     String segmentName,
                                     IncomingNotification.Type type,
-                                    String channel) {
+                                    String channel,
+                                    Long pcn,
+                                    String d,
+                                    CompressType c) {
```

Comment:
It'd be nice to rename these as well
```suggestion
                                    Long previousChangeNumber,
                                    String data,
                                    CompressType compressType) {
```

Reply:
Updated.